standard/statements.md

```diff
@@ -990,20 +1052,29 @@ The target of a `goto` *identifier* statement is the labeled statement with the
 > 
 > class Test
 > {
->     static void Main(string[] args) {
->         string[,] table = {
+>     static void Main(string[] args)
+>     {
+>         string[,] table =
+>         {
 >             {"Red", "Blue", "Green"},
 >             {"Monday", "Wednesday", "Friday"}
 >         };
->         foreach (string str in args) {
+>         foreach (string str in args)
+>         {
 >             int row, colm;
 >             for (row = 0; row <= 1; ++row)
+>             {
 >                 for (colm = 0; colm <= 2; ++colm)
+>                 {
 >                     if (str == table[row,colm])
+>                     {
 >                         goto done;
+>                     }
+>                 }
+>             }
 >             Console.WriteLine("{0} not found", str);
 >             continue;
->         done:
+>           done:
 >             Console.WriteLine("Found {0} at [{1}][{2}]", str, row, colm);
```

Comment:
consider string interpolation:

```suggestion
>             Console.WriteLine($"Found {str} at [{row}][{colm}]");
```

Reply:
Done - and there are no more occurrences of `{0}` :)